cloud/edgecontroller/pkg/devicecontroller/messagelayer/context.go

```diff
@@ -0,0 +1,37 @@
+package messagelayer
```

Comment:
can we move the messagelayer to a "common" folder under pkg ?
in this way edgecontroller and devicecontroller can reuse the same code ?

Reply:
We can move it together with kubeconfig